Sources/StreamChatUI/ChatChannelList/ChatChannelUnreadCountView+SwiftUI_Tests.swift

```diff
@@ -32,6 +31,6 @@ class ChatChannelUnreadCountView_SwiftUI_Tests: XCTestCase {
 
         let view = ChatChannelUnreadCountView.SwiftUIWrapper<CustomUnreadCountView>()
         view.content = .mock(messages: 20)
-        AssertSnapshot(view)
+        AssertSnapshot(view, size: .init(width: 50, height: 50))
```

Comment:
Why the explicit size suddenly?

Reply:
@DominikBucher12 Yes you are actually right, just fixed the issue ✅